.drone.yml

```diff
@@ -2,6 +2,12 @@
 kind: pipeline
 type: kubernetes
 name: build
+trigger:
+  event:
+    - push
+  branch:
```

Comment:
I'd argue that we only want this to run on a tag event and create based off the Drone Tag...?